golang/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/momentohq/client-sdk-go/momento"
+)
+
+func main() {
+	var AuthToken = os.Getenv("MOMENTO_AUTH_TOKEN")
+	const (
+		CacheName             = "cache"
+		ItemDefaultTtlSeconds = 60
+	)
+
+	if AuthToken == "" {
+		log.Fatal("Missing required environment variable MOMENTO_AUTH_TOKEN")
+	}
+
+	// Initializes Momento
+	client, err := momento.SimpleCacheClient(&momento.SimpleCacheClientRequest{
+		AuthToken:         AuthToken,
+		DefaultTtlSeconds: ItemDefaultTtlSeconds,
+	})
+	if err != nil {
+		panic(err)
+	} else {
```

Comment:
panic will exit the program so you can actually move this entire code block outside the `else` makes it easier to read